src/deploy/osp_deployer/settings/config.py

```diff
@@ -240,6 +240,34 @@ def __init__(self, settings_file):
             self.enable_instance_ha = False
         self.overcloud_nodes_pwd = deploy_settings['overcloud_nodes_pwd']
 
+        dellnfv_settings = self.get_settings_section(
+            "Dell NFV Settings")
+        self.ovs_dpdk_enable = dellnfv_settings[
+            'ovs_dpdk_enable']
+        self.ovs_dpdk_role_list = dellnfv_settings[
+            'ovs_dpdk_role_list']
+        self.ovs_dpdk_policy = dellnfv_settings[
+            'ovs_dpdk_policy']
+        self.enable_ovs_dpdk = False
+        if self.ovs_dpdk_enable.lower() == 'false':
+            pass
+        elif self.ovs_dpdk_enable == 'Tenant-networks-only':
+            self.enable_ovs_dpdk = True
+            self.ovs_dpdk_mode = 1
+        elif self.ovs_dpdk_enable == 'Tenant-and-External-networks':
+            self.enable_ovs_dpdk = True
+            self.ovs_dpdk_mode = 2
+        else:
+            raise AssertionError('Only supported values for '
+                                 'ovs_dpdk_enable are false, '
+                                 'Tenant-networks-only and '
+                                 'Tenant-and-External-networks')
+        if self.enable_ovs_dpdk:
+            logger.info("OVS_DPDK is enabled with mode " +
+                        str(self.ovs_dpdk_mode) + ".")
+        else:
+            logger.info("OVS_DPDK is disabled.")
+

```

Comment:
enable_ovs_dpdk  & ovs_dpdk_enable are essentially the same thing, could we just use ovs_dpdk_enable ?
if self.enable_ovs_dpdk can change to self.ovs_dpdk_enable.lower() != 'false':

Reply:
ovs_dpdk_enable can have three states, it's not a boolean. It can accept Tenant-networks-only which is mode 1, Tenant-and-External-networks which is mode 2 or false which disable the feature. That way it exposes the feature in a more user-friendly way. On the other hand, enable_ovs_dpdk determines only if the feature is enabled or not,regardless of the mode. Besides, this one is a boolean.